ui-tests/src/test/java/org/openmrs/reference/page/AttachmentsPage.java

```diff
@@ -0,0 +1,48 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page {
```

Comment:
well this is a good catch @kaweesi  i see reference application depends on distro

Reply:
> i see reference application depends on distro

@HerbertYiga I think you meant qaframework module depends on distro